### PR TITLE
[backend/frontend] Alignment of entity fields in creation and update (#2817)

### LINKED ISSUE #2817: Some entity fields are not aligned in creation and update
## Description

Some entity fields are not aligned in creation and update :

- Note - Creation : external references missing
- Opinion - Creation : external references missing
- Vulnerability - Creation : external references missing
- Feedback - Creation : confidence, object label, external references missing
- Incident - Creation : object label missing
- Incident - Update : confidence missing
- Event- Creation : object label missing
- Sector - Creation : object label missing
- Administrative Area - Creation : object label missing
- City - Creation : object label missing
- Country - Creation : object label, external references missing
- Position - Creation : object label missing
- Region - Creation : object label, external references missing
- Indicator- Creation : score missing
- Infrastructure - Creation : first seen, last seen, kill chain phases missing
- Attack pattern - Creation : external references missing
- Course of action - Creation : external references missing

## Environment

1. OS : Windows 11
2. OpenCTI version: 5.5.3
3. OpenCTI client: frontend
4. Other environment details:

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. go to the dedicated form

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
### Proposed changes

* Align some fields in creation and update
* Add a subscription for entity setting management

### Related issues

* [2817](https://github.com/OpenCTI-Platform/opencti/issues/2817)

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/cases/feedbacks/FeedbackEditionOverview.tsx [hunk 1]
```diff
@@ -273,24 +273,24 @@ FeedbackEditionOverviewProps
       }) => (
         <Form style={{ margin: '20px 0 20px 0' }}>
           <OpenVocabField
-            label={t('Case priority')}
-            type="case_priority_ov"
-            name="priority"
+            label={t('Case severity')}
```
Comment (Archidoit) [inline opencti-platform/opencti-front/src/private/components/cases/feedbacks/FeedbackEditionOverview.tsx:275]: translation to be added?

Comment (RomuDeuxfois) [reply opencti-platform/opencti-front/src/private/components/cases/feedbacks/FeedbackEditionOverview.tsx:275]: Well seen !
Fix

File: opencti-platform/opencti-front/src/private/components/analysis/groupings/GroupingCreation.js [hunk 2]
```diff
@@ -93,8 +93,8 @@ const groupingMutation = graphql`
 
 const groupingValidation = (t) => Yup.object().shape({
   name: Yup.string().required(t('This field is required')),
-  context: Yup.string().required(t('This field is required')),
```
Comment (richard-julien) [inline opencti-platform/opencti-front/src/private/components/analysis/groupings/GroupingCreation.js:96]: Not mandatory in grapqhql

File: opencti-platform/opencti-front/src/private/components/cases/feedbacks/FeedbackCreation.tsx [hunk 3]
```diff
@@ -73,14 +78,18 @@ const feedbackMutation = graphql`
...
-  rating: number;
-  objects: { value: string }[];
-  file: File | undefined;
+  description: string
+  confidence: number
+  rating: number
+  objects: { value: string }[]
+  file: File | undefined
+  objectLabel: Option[]
+  externalReferences: { value: string }[]
```
Comment (richard-julien) [inline opencti-platform/opencti-front/src/private/components/cases/feedbacks/FeedbackCreation.tsx]: External refs not interesting for feedbacks

File: opencti-platform/opencti-front/src/private/components/analysis/groupings/GroupingCreation.js (hunk 2 above)
Comment (RomuDeuxfois) [reply opencti-platform/opencti-front/src/private/components/analysis/groupings/GroupingCreation.js:96]: Fixed

File: opencti-platform/opencti-front/src/private/components/cases/feedbacks/FeedbackCreation.tsx (hunk 3 above)
Comment (RomuDeuxfois) [reply opencti-platform/opencti-front/src/private/components/cases/feedbacks/FeedbackCreation.tsx]: Fixed

**Review by Kedae — APPROVED** [review]

**Pull request merged**